internal/driver/mobile/app/darwin_ios.go

```diff
@@ -195,9 +185,14 @@ func sendTouch(cTouch, cTouchType uintptr, x, y float32) {
 
 	t := touch.Type(cTouchType)
 	if t == touch.TypeEnd {
-		touchIDs[id] = 0
+		// Clear all touchIDs when touch ends. The UITouch pointers are unique
+		// at every multi-touch event. See:
+		// https://github.com/fyne-io/fyne/issues/2407
+		// https://developer.apple.com/documentation/uikit/touches_presses_and_gestures?language=objc
+		for idx := range touchIDs {
```

Comment:
Wouldn't it be quicker to just set `touchIDs` to `nil`?

Reply:
- This involves compiler optimization (see https://github.com/golang/go/issues/5373).
- Set it to nil will cause memory allocation.